Update serverless page (#5747)
DataDog/documentation: content/en/logs/log_collection/_index.md

## Serverless Log collection

> Datadog can collect logs from AWS Lambda. To enable this, refer to the [AWS Lambda integration documentation][20].

Reviewer: ```suggestion
Datadog collects logs from AWS Lambda. To enable this, refer to the [AWS Lambda integration documentation][20].
```

## Datadog Logs Endpoints

Datadog provides logging endpoints for both SSL-encrypted connections and unencrypted connections.

> You should use the encrypted endpoint when possible. The Datadog Agent uses the encrypted endpoint to send logs to Datadog (more information available in the [Datadog security documentation][22]).

Reviewer: ```suggestion
Use the encrypted endpoint when possible. The Datadog Agent uses the encrypted endpoint to send logs to Datadog. More information is available in the [Datadog security documentation][22].
```
